extensions/resume.md

```diff
@@ -214,12 +214,12 @@ Successful `RESUME` attempt from a client with the nick `dan` reconnecting. The
 Here is this successful reconnection seen by `george`, a client that does not have the `draft/resume-0.4` capability enabled:
 
     S: :dan!~old@192.168.0.5 QUIT :Client reconnected (24 seconds of message history lost)
-    S: :dan!~d@127.0.0.1 JOIN #test
+    S: :dan!~old@127.0.0.1 JOIN #test
```

Comment:
Let's be consistent and replace `~old` with `~u` (was only called old because of the old/new username stuff, now it makes no sense to retain that name).